rectangling.qmd

```diff
@@ -362,11 +362,11 @@ You'll learn more about `map_lgl()` in @sec-iteration.
 
 tidyr has a few other useful rectangling functions that we're not going to cover in this book:
 
--   `unnest_auto()` automatically picks between `unnest_longer()` and `unnest_wider()` based on the structure of the list-column. It's a great for rapid exploration, but ultimately its a bad idea because it doesn't force you to understand how your data is structured, and makes your code harder to understand.
+-   `unnest_auto()` automatically picks between `unnest_longer()` and `unnest_wider()` based on the structure of the list-column. It's great for rapid exploration, but ultimately it's a bad idea because it doesn't force you to understand how your data is structured, and makes your code harder to understand.
 -   `unnest()` expands both rows and columns. It's useful when you have a list-column that contains a 2d structure like a data frame, which you don't see in this book.
```

Comment:
The sentence following `unnest()` ends rather abruptly, i.e. "[...], which you don't see in this book". Could you perhaps refer to a resource that uses `unnest()` in more detail? I remember the first time I saw the `unnest()` function in action was in the first version of R4DS when you showed how to iteratively run a linear model on the nested gapminder dataset. In the intro to this second edition of R4DS you mentioned however, that all those modeling parts are removed and you instead refer to the book "Tidy Modeling with R". Maybe that could be mentioned here as well? Sorry for the long comment  :D

Reply:
Hmmm, maybe https://www.tmwr.org/base-r.html#combining-base-r-models-and-the-tidyverse ?

Reply:
Yes that's great, I like it - and it gives people more avenues to explore.